<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>line_page</class>
 <widget class="QMainWindow" name="line_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="line_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Build Line Chart</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="fr_down">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>521</width>
      <height>471</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="fr_up">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 255, 189);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Line Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="x_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>X label name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="y_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Y label name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="color_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First line color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="first_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First Array:</string>
    </property>
   </widget>
   <widget class="QLabel" name="second_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second Array:</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantity_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Elements Quantity:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="f_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="s_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="q_line">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="color_box_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Grey</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purple</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="display_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(166, 183, 255);</string>
    </property>
    <property name="text">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QLabel" name="width_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First line width:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="width_box_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="style_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First linestyle:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="style_box_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>solid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dotted</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dashed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dashdot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="color_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second line color:</string>
    </property>
   </widget>
   <widget class="QLabel" name="style_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second linestyle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="width_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second line width:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="style_box_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>solid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dotted</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dashed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>dashdot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="color_box_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Brown</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Grey</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Purple</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="width_box_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="marker_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First line marker:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marker_box_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>v</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="marker_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second line marker:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="marker_box_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>v</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="size_label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>First marker size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="size_box_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="size_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Second marker size:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="size_box_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
